pkg/addons/default/assets/aws-node.yaml

```diff
@@ -54,6 +54,8 @@ metadata:
 spec:
```

Comment:
For your reference https://github.com/aws/amazon-vpc-cni-k8s/blob/release-1.6/config/v1.6/aws-k8s-cni.yaml